theano/tensor/extra_ops.py

```diff
@@ -12,16 +12,11 @@
 
 class CumsumOp(theano.Op):
     # See function cumsum for docstring
+    __props__ = ("axis")
```

Comment:
this don't make a tuple, this create just a string. use `("axis",)`, the comma is important. That is what make travis fail.